tests/test_api.py

```diff
@@ -148,6 +148,24 @@ def test_newPage_newGState():
     fill(1)
     assert (255, 255, 255, 255) == db._gstate.fillPaint.color
 
+def test_newPage_dimensions_arguments():
+    from drawbot_skia.drawing import default_canvas_dimensions
+    db = Drawing()
+    db.rect(0, 0, 300, 300) # the dimensions are set only after drawing starts
+    assert (db.width(), db.height()) == default_canvas_dimensions, (
+        "No args and first page - dimensions differ"
+        )
+    db.newPage(600, 450)
+    db.rect(0, 0, 300, 300)
+    assert (db.width(), db.height()) == (600, 450), (
+        "Args setting dimensions - dimensions differ"
+        )
+    db.newPage()
+    db.rect(0, 0, 300, 300)
+    assert (db.width(), db.height()) == (600, 450), (
+        "No args but not first page - dimensions differ" 
+        )
```

Comment:
Please also check that calling `newPage(123)` and `newPage(height=123)` raise errors.

```python
with pytest.raises(...):
    ...code that should raise an exception...
```